v3/setup/mesos-credentials.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -x
+
+source /etc/environment
+if [ "${NODE_ROLE}" != "control" ] && [ "${NODE_ROLE}" != "worker"]; then
+    exit 0
+fi
+
+# ensure that we have sane defaults for credentials
+etcdctl get /mesos/config/username
+if [[ $? = 4 ]]; then
+    etcdctl set /mesos/config/username ethos
+fi
+etcdctl get /mesos/config/password
+if [[ $? = 4 ]]; then
+    etcdctl set /mesos/config/password password
```

Comment:
I guess that I could just exit the script with a very "loud" warning? This will prevent the credential files from being written and the changes that I added in the `ExecStartPre` steps of the units will result in failures.